lib/main.js

```diff
@@ -0,0 +1,156 @@
+'use babel'
+
+/* @flow */
+
+import Path from 'path'
+import { CompositeDisposable } from 'atom'
+
+type Linter$Provider = Object
+
+module.exports = {
+  config: {
+    executablePath: {
+      type: 'string',
+      default: Path.join(__dirname, '..', 'node_modules', 'jshint', 'bin', 'jshint'),
+      description: 'Path of the `jshint` executable'
```

Comment:
Again, this needs to be the `jshint` _Node_ script, not a system executable. Maybe something like this?
> 'Path to the `jshint` Node script'